### PR TITLE
Feature/334/335/view case progress chart

### PR DESCRIPTION
**Pull request by caoimheb**

### CODE REVIEW

**Comment by andrew-js-wright** [comment]
some standard issues being flagged by travis.

**Comment by caoimheb** [comment]
Travis error `Error: Cannot find module './data/get-individual-caseload-progress'`. Module is placeholder for DB query file in other branch

File: app/views/case-progress.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+{% include "includes/breadcrumbs.html" %}
+
+{% include "includes/validation-error-messages.html" %}
+
+<h2 class="heading-large">
+    <span class="heading-secondary">
+        {{ subTitle | title }}
+    </span>
+    {{ title | title }}
+</h2>
```
Comment (andrew-js-wright) [inline app/views/case-progress.html]: A nice improvement here might be to extract this into a partial like the breadcrumbs, as we're now using it multiple places, and will likely need it on most pages to come.

Comment (caoimheb) [inline app/views/case-progress.html]: Good shout. Updated

File: app/routes/capacity-route.js [hunk 2]
```diff
@@ -14,7 +14,7 @@ module.exports = function (router) {
       return res.render('capacity', {
         title: result.title,
         subTitle: result.subTitle,
-        subnav: getSubNav(id, organisationLevel, req.path),
+        subNav: getSubNav(id, organisationLevel, req.path),
```
Comment (andrew-js-wright) [inline app/routes/capacity-route.js]: 👍

**Pull request merged**